docs/pages/introduction/why-not-expo.mdx

```diff
@@ -3,108 +3,110 @@ title: Limitations
 
-<Collapsible summary="Your app cannot target only children under 13 years old without customizing native dependencies">
-
-Both [Apple](https://developer.apple.com/app-store/review/guidelines/#kids) and [Google](https://support.google.com/googleplay/android-developer/answer/9285070?hl=en) provide strict guidelines for any apps that specifically target children under a particular age. One of these guidelines states that certain ad libraries, such as Facebook's Audience Network, cannot be used in the app.
+</Collapsible>
 
-Apps built with `expo build:ios|android` [contain code for the entire Expo SDK](https://expo.fyi/managed-app-size), you cannot customize the native dependencies, including Facebook's Audience Network library, so if you build your app this way you cannot designated it as "designed primarily for children under 13" in the App Store or Play Store, _even though this code is never run unless you explicitly call it_.
+<Collapsible summary="Your app cannot target children under 13 years old without customizing native dependencies">
 
-Apps built with `eas build -p ios|android` [include only your app's explicit native dependencies](https://blog.expo.dev/expo-managed-workflow-in-2021-d1c9b68aa10), and so this limitation does not apply if you use EAS Build. [Learn about how to use it](/build/introduction).
+Apps built with `expo build:ios|android` contain code for the entire Expo SDK. Unfortunately, you cannot customize the native dependencies, including Facebook's Audience Network library. So if you build your app this way, you cannot designate it as "designed primarily for children under 13" in the app stores, even though this code is never run unless you explicitly call it.
 
 </Collapsible>
 
-> We are either actively working on or planning to build solutions to all of the limitations listed above, and if you think anything is missing, please bring it to our attention by posting to our [feature requests board](https://expo.canny.io/feature-requests) or the [forums](https://forums.expo.dev/).
+## Bare workflow
 
-## Limitations of the bare workflow
+> If you'd like to continue getting the same development experience that you have with Expo Go, but also use custom native libraries or custom native code, you should check out the [Development builds]()
 
-In the bare workflow, we have full access to the underlying native projects and any native code. It's a "bare" native project with React Native and one or more packages from the Expo SDK installed. Anything that you can do in a native project is possible here.
+As similar to [Development builds], bare workflow also provides access to the underlying native projects and any native code. It's a "bare" native project where developers make direct changes to their native `android` and `ios` project directories rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild/).
```

Comment:
Realizing that I may have suggested so many changes that, if you were to accept them all, not much of this section may be left. In general, I think we could summarize bare workflow under EAS Build as something to the effect of "EAS Build still works if you check in your ios and android native project folders, but prebuild can no longer sync changes from your Expo config to the native projects once you do that; you'll have to configure them on your own with native tools like Xcode and Android Studio."